data/core/editor/help.cfg

```diff
@@ -465,12 +459,12 @@ In scenario mode, the button accesses an editor for individual schedules for <re
     # po: the “Player 1”, “Player 2”, ... list is translated using "Player $side_num" in the wesnoth-editor textdomain
     text= _ "The editor palette contains the applicable items you may use with the currently selected tool. For example, the Paint tool will display a full list of all available terrains, and the unit tool will provide a list of available units. When using the Starting Locations Tool, the palette changes to a list of “Player 1”, “Player 2”, etc.
 
-<bold>text='Filter'</bold>
+<bold>Filter</bold>
 
 There is a filter function to show only a subset of the available items — this is the leftmost of the four buttons at the top of the palette, and the graphic changes depending on what is selected. Examples:" +
-        "<img>src=icons/terrain/terrain_group_all_30.png align=left box=yes</img>" + _ "Show all kinds of terrain" +
-        "<img>src=icons/terrain/terrain_group_water_deep_30.png align=left box=yes</img>" + _ "Show only water terrains" +
-        "<img>src=icons/terrain/terrain_group_village_30.png align=left box=yes</img>" + _ "Show only villages"
+        "<table/><img src=icons/terrain/terrain_group_all_30.png align=left box=yes /></col><col>" + _ "Show all kinds of terrain</col></row>" +
```

Comment:
Sorry I missed this, but the mark-up here should have been added separately from the translated string. I'll probably make a PR to resolve this shortly.